docker/Dockerfile.dhtnode

```diff
@@ -0,0 +1,25 @@
+# Duplicate definition of DIST before FROM is needed to be able to use it
+# in docker image name:
+ARG  DIST=xenial
+FROM sociomantictsunami/dlang:$DIST-v5 as builder
+# Copies the whole project as makd needs git history:
+COPY . /project/
+WORKDIR /project/
+# Redefine arguments as env vars to be used inside build.sh script:
+ARG DIST=xenial
+ARG DMD=1.081.*
+ENV DMD=$DMD DIST=$DIST
+RUN docker/build.sh
+
+ARG DIST=xenial
+# For now plain ubuntu image is used as base for simplicity. It certainly can be
+# optimized to bare minimum but right now it is not important:
```

Comment:
Just for the records since we talked about this recently with @federico-cuello-sociomantic , in some future we could aim to use https://www.alpinelinux.org/. But this is extremely low prio for now.

Reply:
Just to be clear that I'm not endorsing, but the idea of having other options and not just Ubuntu.